### PR TITLE
fail test

### PR DESCRIPTION
**Pull request by Leonz5288**
Related issue = #

[[Click here for the format server]](http://kun.csail.mit.edu:31415/)

----

### CODE REVIEW

**Comment by Leonz5288** [comment]
#test

**Review by Leonz5288 — COMMENTED** [review]
as

**Review by Leonz5288 — COMMENTED** [review]
#test

**Review by k-ye — COMMENTED** [review]
#test

File: cmake/TaichiCore.cmake [hunk 1]
```diff
@@ -1,9 +1,9 @@
-option(USE_STDCPP "Use -stdlib=libc++" OFF)
-option(TI_WITH_CUDA "Build with the CUDA backend" ON)
-option(TI_WITH_OPENGL "Build with the OpenGL backend" ON)
-option(TI_WITH_CC "Build with the C backend" ON)
+# option(USE_STDCPP "Use -stdlib=libc++" OFF)
+# option(TI_WITH_CUDA "Build with the CUDA backend" ON)
```
Comment (k-ye) [inline cmake/TaichiCore.cmake:2]: #test

File: cmake/TaichiCore.cmake [hunk 2]
```diff
@@ -1,9 +1,9 @@
...
+# option(TI_WITH_OPENGL "Build with the OpenGL backend" ON)
+# option(TI_WITH_CC "Build with the C backend" ON)
```
Comment (k-ye) [inline cmake/TaichiCore.cmake:4]: aaaa

**Pull request merged**